src/main/java/com/epam/aidial/cfg/dao/model/TimeTrackableEntity.java

```diff
@@ -0,0 +1,34 @@
+package com.epam.aidial.cfg.dao.model;
+
+import jakarta.persistence.MappedSuperclass;
+import jakarta.persistence.PrePersist;
+import jakarta.persistence.PreUpdate;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.envers.Audited;
+
+import java.time.Instant;
+
+@Getter
+@Setter
+@Audited
+@MappedSuperclass
+public abstract class TimeTrackableEntity extends AbstractEntity<String> {
+
+    private Long createdAt;
```

Comment:
add comment or change name. it's not obvious what unit used for time (s/ms/microsec/nanosec)